src/analysis/inner_product_space/positive.lean

```diff
@@ -124,3 +126,198 @@ end
 end complex
 
 end continuous_linear_map
+
+namespace linear_map
+
+open linear_map
+
+variables {V : Type*} [inner_product_space ℂ V]
+
+local notation `e` := is_symmetric.eigenvector_basis
+open_locale big_operators
+
+/-- `T` is (semi-definite) positive if `∀ x : V, ⟪x, T x⟫_ℂ ≥ 0` -/
+def is_positive (T : V →ₗ[ℂ] V) :
+  Prop := ∀ x : V, 0 ≤ ⟪x, T x⟫_ℂ.re ∧ (⟪x, T x⟫_ℂ.re : ℂ) = ⟪x, T x⟫_ℂ
```

Comment:
I think that you should make this definition work in the real case too (like for continuous linear maps above) by explicitly asking for `is_symmetric`, and then prove that in the complex case the condition you have here is enough. More generally, while I'm completely fine with having this special case definition because translating to continuous linear maps all the time, one of the first lemma you prove should be the fact that it reduces to positivity of `T.to_continuous_linear_map`, and then use that to build up the API

Reply:
did this and made all the results work in the real case too